recipes/optunahub/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "optunahub" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/optunahub-{{ version }}.tar.gz
+  sha256: 6fc08afcdb9ca35dc1badfa4a8152c7645f3c25f125d011e08ac897f13152d0a
```

Comment:
I confirmed that the hash value matches the one from https://pypi.org/project/optunahub/#optunahub-0.2.0.tar.gz.